### PR TITLE
FIrst PR dev to main

### PR DESCRIPTION
**Pull request by lehmeti**
Just for training of PR handling.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -129,4 +129,6 @@ dmypy.json
 .pyre/
 
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+
```
Comment (2Who) [inline .gitignore:133]: Sehr gut 👍

File: .gitignore [hunk 2]
```diff
@@ -129,4 +129,6 @@ dmypy.json
...
+/venv
```
Comment (2Who) [inline .gitignore:134]: Sehr gut 👍

Comment (2Who) [inline .gitignore]: Sehr gut 👍

**Review by 2Who — APPROVED** [review]
Perfekt!

**Review by steibern — APPROVED** [review]

**Pull request merged**